org.mozilla.Firefox.appdata.xml

```diff
@@ -11,7 +12,7 @@
   <url type="homepage">https://www.mozilla.org/en-US/</url>
   <screenshots>
     <screenshot type="default">
-      <image>http://www.codeblocks.org/images/stories/imagebrowser/scr1.png</image>
+      <image>https://raw.githubusercontent.com/endlessm/firefox-flatpak/master/screenshot.png</image>
```

Comment:
The URL should rather be: https://d3lapyynmdp1i9.cloudfront.net/screenshots/org.mozilla.Firefox/C/org.mozilla.firefox-screenshot1.jpg